django_app/tests/conftest.py

```diff
@@ -26,6 +28,34 @@
 logger = logging.getLogger(__name__)
 
 
+@pytest.fixture()
```

Comment:
cut and paste from the now deleted client.py as this is the only place that where it is used